frontend/src/component/demo/Demo.tsx

```diff
@@ -18,34 +20,47 @@ const { value: storedProgress, setValue: setStoredProgress } =
 
 export const Demo = () => {
     const { uiConfig } = useUiConfig();
-    const [loaded, setLoaded] = useState(false);
+    const [welcomeOpen, setWelcomeOpen] = useState(
+        storedProgress.welcomeOpen ?? true
```

Comment:
Will this reset every time we deploy a new version? I don't remember how storedProgress is persisted, but I just thought asking the question

Reply:
It's persisted on local storage, I don't think deploying new versions of Unleash would affect it. 

This local storage set is also versioned `Tutorial:v1`, so if we later release a new version of the tutorial and want to reset it for end-users we just need to increase this version.

Users have the choice of re-entering the welcome screen at any point through the topic list by clicking "View demo link again" at the bottom of the list:
![image](https://user-images.githubusercontent.com/14320932/233414647-2e3a3223-6299-4787-929e-6c7db0b164c7.png)
Clicking any of the topics at any point would start that topic as well.